app/src/test/java/org/simple/clinic/scanid/ScanSimpleIdUpdateTest.kt

```diff
@@ -162,4 +162,18 @@ class ScanSimpleIdUpdateTest {
             hasEffects(OpenPatientSearch(null, "12345612345612"))
         ))
   }
+
+  @Test
+  fun `when entered code is not a short code, then open patient search`() {
+    val shortCode = "12345678901234"
```

Comment:
~Can we rename `shortCode` to `enteredCode` here?~ I have a renaming PR coming up for this package will include this as well.